src/main/java/io/cdap/e2e/pages/actions/CdfControlCenterActions.java

```diff
@@ -0,0 +1,402 @@
+  }
+
+  /**
+   * Check for the displayed message when sorted by Newest.
+   */
+  public static void clickOnNewestOptionMessage() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.newestOptionMessage);
+  }
+
+  /**
+   * Check for the displayed message when sorted by Oldest.
+   */
+  public static void clickOnOldestOptionMessage() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.oldestOptionMessage);
+  }
+
+  /**
+   * Check for the displayed message when sorted by Z To A.
+   */
+  public static void clickOnZToAOptionMessage() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.zToAOptionMessage);
+  }
+
+  /**
+   * Check for the displayed message when sorted by A To Z.
+   */
+  public static void clickOnAToZOptionMessage() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.aToZOptionMessage);
+  }
+
+  /**
+   * Click on the sort by dropdown.
+   */
+  public static void selectSortDropdown() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.sortDropdown);
+  }
+
+  /**
+   * Select the dropdown with value.
+   */
+  public static void selectSortDropdownOptionValue(String option) {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.selectSortDropdownValue(option));
+  }
+
+  /**
+   * Click on the plus add button to add a tag.
+   */
+  public static void clickOnAddTagIcon() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.addTagIconDatasetEntity);
+  }
+
+  /**
+   * Enter the value of the tag.
+   */
+  public static void enterValueTagInputField(String value) {
+    WebElement pluginPropertyInput = CdfControlCenterLocators.enterAddTagIconDatasetEntityValue();
+    String valueFromPluginPropertiesFile = PluginPropertyUtils.pluginProp(value);
+    if (valueFromPluginPropertiesFile == null) {
+      ElementHelper.sendKeys(pluginPropertyInput, value);
+      return;
+    }
+    ElementHelper.sendKeys(pluginPropertyInput, valueFromPluginPropertiesFile);
+  }
+
+  /**
+   * Check for the count when user adds the tag.
+   */
+  public static void tagCountIncreasesDataset() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.tagCountIconIncreaseDatasetEntity);
+  }
+
+  /**
+   * Click on close tab of tags in Dataset Entity Details Page.
+   */
+  public static void clickOnCloseIcon() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.closeTagIconDatasetEntity);
+  }
+
+  /**
+   * Check for the count when user removes the tag.
+   */
+  public static void tagCountDecreasesDataset() {
+    ElementHelper.clickOnElement(CdfControlCenterLocators.tagCounDecreaseIconDatasetEntityMessage);
+  }
+
+  /**
+   * Click on Search tab in control center.
+   */
+  public static void clickOnSearchTagControlCenter(String searchedText) {
+    ElementHelper.sendKeys(CdfControlCenterLocators.searchTabControlCenter, searchedText);
+    ElementHelper.clickOnElement(CdfControlCenterLocators.searchTabControlCenter);
+  }
```

Comment:
rename this method to make it clear what it does. probably to `enterTextInSearchBarInControlCenter`

Reply:
Renamed it to above .